manifests/example-config.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   name: config
 data:
   config: |
+    protocol: "bgp"
```

Comment:
Can you make this field per-address-pool? In my cluster, I want some address pools advertised over BGP, and others advertised over ARP :)

Another open question: for a single address pool, does it make sense to advertise several protocols? As in, `protocol: ["bgp", "arp"] ? I'm not sure if it makes sense, but I don't know. WDYT?